lib/package.gi

```diff
@@ -1626,12 +1635,14 @@ InstallGlobalFunction( LoadPackage, function( arg )
 ##
 #F  LoadAllPackages()
 ##
-InstallGlobalFunction( LoadAllPackages, function()
-    if ValueOption( "reversed" ) = true then
-    	List( Reversed( RecNames( GAPInfo.PackagesInfo ) ), LoadPackage );
-    else
-    	List( RecNames( GAPInfo.PackagesInfo ), LoadPackage );
-    fi;	
+    InstallGlobalFunction( LoadAllPackages, function()
+        SuspendMethodReordering();        
+        if ValueOption( "reversed" ) = true then
+            List( Reversed( RecNames( GAPInfo.PackagesInfo ) ), LoadPackage );
+        else
+            List( RecNames( GAPInfo.PackagesInfo ), LoadPackage );
+        fi;	
+        ResumeMethodReordering();        
```

Comment:
This change of indentation increases the diff needlessly... It seems the only actual changes was that a call to `SuspendMethodReordering` was inserted?

Reply:
Oh and also ResumeMethodReordering

Reply:
Thanks, will fix.